.github/workflows/custom-test.yml

```diff
@@ -8,15 +8,16 @@ on:
         required: true
         type: string
 
-permissions:
-  id-token: 'write' # For signing with the GitHub workflow identity
+permissions: {}
 
 env:
   METADATA_URL: ${{ inputs.metadata_url }}
   IDENTITY: ${{ github.server_url }}/${{ github.repository }}/.github/workflows/custom-test.yml@${{ github.ref }}
 
 jobs:
   sigstore-python:
+    permissions:
```

Comment:
This would need to be replicated to all jobs in this file, as they all will request an id-token to use the ephemeral key flow to sign.

Reply:
Whoops, thanks for catching that! I missed that there were other individual jobs that did signing. Will fix now.

Reply:
Done -- FWICT only two other jobs do signing, so I added it to each.